lib/cdo/hamburger.rb

```diff
@@ -78,7 +73,6 @@ def self.get_visibility(options)
       hamburger_class: hamburger_class,
       show_teacher_options: show_teacher_options,
       show_student_options: show_student_options,
-      show_signed_out_options: show_signed_out_options,
```

Comment:
While this is removed, I think it may still be helpful to define a local variable or function to determine if the user is a teacher or a student, or neither so that you can easily replace all uses of `options[:user_type] == "teacher" || options[:user_type] == "student"` below

Reply:
Good call, just added!